communication/marketing-team/multichannel-communications.md

```diff
@@ -0,0 +1,82 @@
+# Multichannel communications
+
+The goal of this is to provide a clearer way for community members to share and consume Kubernetes information.
+
+## Motive
+Our motive is as follows:
+
+* Provide all Kubernetes contributors with a dependable experience of communication to and from the project
+* Find ways of adding our human touch to potentially overwhelming quantities of information
+* Provide clear expectations on what’s important, urgent, or otherwise
+* Build process like we build Kubernetes, so that our contributors understands it, and it works well at scale
+
+## Outbound and inbound communications
+
+All outbound communications will have a single source of truth in kubernetes-dev@googlegroups.com. 
+
+Justification for this canonical location:
+* Per 2018 and 2019 survey results, it has grown in popularity as the one place to check for contributor news
+* Highlighted in the contributor guide, SIG READMEs, governance documentation, and many other places
+
+All other communication channels will point to that canonical location because we want to:
+
+* avoid having multiple places with the same information
+* avoid confusion on where to get the right information
+* create an easier process for information sharing
+
+## Mapping Type to Tier
+
+We have identified four (4) different tiers of information:
+
+* Action Required
+    * Participation by contributors is required
+    * Example 1: The Contributor Survey
+    * Example 2: Code freeze is at $date, get your code in!
+        * Other release information should go through the release team
+* Priority Information
+    * Critical information regarding the project's sustainability
+    * Example 1: Voting in the steering committee election
+    * Example 2: Changes in workflows, issue triaging, etc
+* Feedback Appreciated
+    * When participation is helpful, but not required by everyone in order to be successful
+    * Example: Kubernetes Enhancement Proposals (KEPs)
+* Important Information
+    * When we want others to see new updates, but it does not necessarily have an action item
```

Comment:
I'm getting snagged on the question if people will remember the difference between `Priority Information` and `Important Information`. I don't like the pedantic arguments it may lead to. Is there a way to give this more of a lighter tone?

```suggestion
* Updates
    * Information that is not urgent or priority, but will be interesting to others and worthwhile to highlight
```

Other ideas:

* Noteworthy
* In case you missed it (ICYMI)
* kgossip (please don't choose this)
* Did you know
* News